packages/campaign-form/components/CampaignForm/index.jsx

```diff
@@ -37,20 +37,25 @@ const Card = styled.div`
   display: flex;
   flex-direction: column;
   background: ${white};
-  border: 1px solid ${gray};
+  border: 1px solid ${grayLight};
   box-sizing: border-box;
   border-radius: ${borderRadius};
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.08);
   width: 100%;
   padding: 0 16px;
-  text-align: left;
+`;
+
+const HeadlineContainer = styled.div`
```

Comment:
There's a way to apply this without creating an extra div 🙂 
```suggestion
const Headline = styled(Text)`
```
And then you replace `<Text type="h2">` by `<Headline type="h2">`

Reply:
This was a spot where I was totally guessing how to make this work. Thanks for showing me the proper way!